app/models/manageiq/providers/ansible_runner_workflow.rb

```diff
@@ -2,17 +2,49 @@ class ManageIQ::Providers::AnsibleRunnerWorkflow < Job
   def self.create_job(env_vars, extra_vars, role_or_playbook_options,
                       hosts = ["localhost"], credentials = [],
                       timeout: 1.hour, poll_interval: 1.second, verbosity: 0, become_enabled: false)
-    options = job_options(env_vars, extra_vars, role_or_playbook_options, timeout,
-                          poll_interval, hosts, credentials, verbosity, become_enabled)
-    super(name, options)
+    super(name, role_or_playbook_options.merge(
```

Comment:
Do we know if there was anything in this `role_or_playbook_options` that we were ignoring previously that will now be included?

Reply:
That's a good question...I'll see

Reply:
None of the callers pass anything other than the expected options.  Even so, the launch_runner methods only pull out the options they care about, so extra stuff would be ignored.